app/models/champ.rb

```
@@ -102,8 +102,11 @@ def serialize_datetime_if_needed
       rescue
         nil
       end
-
       self.value = date
+    elsif /^\d{2}\/\d{2}\/\d{4}\s\d{2}:\d{2}$/ =~ value # old browsers can send with dd/mm/yyyy hh:mm format
+      self.value = DateTime.parse(value, "%d/%m/%Y %H:%M").strftime("%Y-%m-%d %H:%M")
+    elsif !(/^\d{4}-\d{2}-\d{2}\s\d{2}:\d{2}$/ =~ value) # a datetime not correctly formatted should not be stored
+      self.value = nil
```

Ça affiche une erreur à l'utilisateur ou pas ?

---

Si le champs est "required" oui

---

Ok, mais du coup si j'écris "jeudi 3 janvier" et que c'est pas un champ required, ça va silently failed. Pour moi c'est pas acceptable

---

C'est déjà le cas. On ne change pas le comportement existant. C'est peut-être un problem, mais hors scope de cette PR.

---

OK, du coup je veux bien une issue (d'ailleurs un utilisateur m'a déjà communiqué le pb, je le rajouterai à l'issue)

---

J'ai crée l'issue https://github.com/betagouv/tps/issues/1377